shelley/design-spec/delegation_design_spec.tex

```diff
@@ -545,276 +573,197 @@ \section{Design of Delegation}
-                     , text width = 5em
-                     , font = \footnotesize
-                     , >={Latex[width=0.5em, length=0.5em]}
-                     , every node/.style = { rectangle
-                                         , rounded corners
-                                         , draw = black
-                                         , align = center
-                                         , minimum height = 4em }
-                     ]
-
+Delegation is a separation of the control over the movements of funds and the
+rights (and obligations) in the PoS protocol that are associated with those
+funds. We achieve this separation by modelling it in the address structure. We
+will distinguish between \emph{value addresses} that determine how funds can be
+spent, and \emph{stake addresses} that define if and how those funds take part
+in the PoS protocol. Actual coins will belong to an (overall) \emph{address},
+consisting of both a value address and a stake address.
+
+We support multi signature (multi-sig) schemes, for payments as well as for
+delegations. We do this by allowing value addresses and script addresses to use
+either keypairs, or scripts for authorisation, and implementing a simple
+scripting language to describe multi-sig schemes. Introducing multi-sig in this
+way has the benefit of naturally generalising when we will later introduce more
+powerful scripting languages, such as Plutus and Marlowe.
+
+Participating in the PoS protocol requires two steps:
+\begin{description}
+\item[Using a registered stake address] Users can post certificates to the chain
+  to register a stake address. This will allow them to delegate funds associated
+  with that address, and also automatically set up a corresponding \emph{reward
+    account}, where the system will accumulate rewards for delegating funds from
+  that stake address.
+\item[Delegating from that stake address to a registered \emph{stake pool}] All
+  blocks in Cardano-Shelley will be produced by a set of stake pools that need
+  to be registered on the chain, by posting an appropriate certificate.
+  Individual stakeholders can \emph{delegate} funds from each of their
+  registered stake addresses to a pool of their choosing. Stake in an address
+  that delegates to a pool counts to the stake of that pool in the leader
+  election. The pool will be rewarded for block production, and those rewards
+  will automatically be distributed to the apropriate reward addresses.
+
+  Note that this does not restrict an individual stakeholder wanting to use
+  their own stake to produce blocks (``self staking''). Such users should
+  register a \emph{private stake pool}, and delegate their own funds to that
+  pool. This uniform architecture, not distinguishing between those stakeholders
+  that are using their stake directly and those that are delegating, reduces the
+  overall complexity of the system significantly.
+\end{description}
```

Comment:
Perhaps worth noting here something like:

> Delegation is optional, but it is the only way to exercise the stake rights to take part in the PoS protocol and to earn rewards.

Reply:
Say "Proof-of-Stake" rather than PoS (unfortunate clash of acronym)